src/test/java/tests/MultiModelTest.java

```diff
@@ -23,7 +23,7 @@ public static void setup() {
         UI.status = new StatusUIStub();
     }
 
-    MultiModel multiModel = new MultiModel(new Preferences(true));
+    MultiModel multiModel = new MultiModel(mock(Preferences.class));
```

Comment:
The mocks are used on tests that are not trying to test `Preferences` but need it to test other classes. E.g. all tests in `MultiModelTests` don't use `Preferences` but need it for the constructor.

`mockito` fakes the behavior of a class and doesn't use the original class's code unless partial mocking features like `spy` are used.